Event_spec.md

```diff
@@ -4,12 +4,13 @@ The port call event format is an implementation of the funtional definitions for
  		
 This specification is in active development, and comments from the port community are welcome.
 
-## MovementId, BerthVisitId and ServiceId
+## MovementId, BerthVisitId, ServiceId and OrganisationPortcallId
 
 The event system allows for three additional identifiers in addition to portcall id's:
 * Movement identifier: this identifies a movement, which is a ship traveling from one location to another inside a portcall
 * Berth visit identifier: this identifier a berth visit, which is a ship being alongside a single berth
 * Service identifier: this identifies a single service like bunkering
+* Organisation port call identifier: this identifier a visit, which is a ship its stay within a single harbour
```

Comment:
identifier -> identifies, also in berth visit

`this identifies a visit, which is a ship its stay within a single port. This is an internal identifier for an organization, independant of the portcall id which is the identifier of the local port authority`

Reply:
good one 👍